src/main/index.html

```diff
@@ -22,41 +22,51 @@
     <title>Twardy Fitness Center</title>
   </head>
   <body>
-    <div id="section-1">
+    <section id="section-1" class="active">
       <div id="header">
```

Comment:
A może tu dać <header> zamiast <div>?

Reply:
Header pewnie bedzie musial wyjsc na section. I jak bedziesz commitowal swoje zmiany to mozesz to edytowac wedle woli. Poki co zrobilem po prostu div'a ktory zrobi za pewien element abstrakcji i zrobic miejsce w kodzie gdzie wrzucimy header tag. Na ten moment nie ma to znaczenia ale znajac zycie zmienimy to potem tak jak mówisz